examples/5_secret_storage.py

```diff
@@ -451,7 +462,7 @@ def noop_handler(*args, **kwargs):
     rental = RentalBotClient(bot)
 
     print("...basic rental actions...")
-    await basic_rental_users(bot, rental, team)
+    await basic_rental_users(rental, team)
 
     print("...multiple users try to reserve...")
     await concurrent_rental_users(bot, rental, team)
```

Comment:
i suppose we could remove `bot` from this function args too?

Reply:
done